### DISCUSSION TITLE
Manubot: Manubot is a workflow and set of tools for the next generation of scholarly publishing. Write your manuscript in markdown, track it with git, automatically convert it to .html, .pdf, or .docx, and deploy it to your destination of choice.

### DISCUSSION THREAD

**Discussion by VictorVenema** [singlesourcepub/community#32] [General]
I just found Manubot. Sounds like a quite similar workflow. What do you think about it?
https://manubot.org

**Comment by xldrkp** [comment]
Hi @VictorVenema, thanks for mentioning manubot. In our project Modern Publishing we knew quite early about it and were fascinated by what it can produce. It is a tool for authors that are familiar with GitHub and a tech-savvy workflow in research and writing. You can find that more often in STEM disciplines. As we were dealing with social sciences and humanities in our project we had to flatten the learning curve for authors and editors and reduce complexity.

**Reply by johav** [reply]
sounds great, do you have documentation for the flattened workflow that is citable? Would like to share it in our courses.

**Reply by VictorVenema** [reply]
I do not know much more than that they exist.

**Reply by xldrkp** [reply]
@johav Please have a look at our project website https://oa-pub.hos.tuhh.de/de/ where we put down a lot of our experiences. There are still steps to be taken to make our system more usable. If you like, let's get in touch about the courses you've just mentioned.

**Comment by agitter** [comment]
I'm one of the Manubot developers and very belatedly found this discussion and cool community. I wanted to point to a [contributing guide](https://github.com/greenelab/covid19-review/blob/master/CONTRIBUTING.md) that one Manubot project created that was intended to be more instructive about how to edit or review changes to content through the GitHub web interface. It was aimed at a less tech-savvy audience, such as biologists or clinicians, contributing to that project.

**Reply by xldrkp** [reply]
Hi @agitter, great you found our communnity here! Would you like to join our monthly community call next Tuesday, 12:00 CEST and show us around the guide you mention?

**Reply by agitter** [reply]
Thanks for the invite, but unfortunately I won't be able to join calls at 12:00 CEST based on where I'm located.
